components/camel-qute/src/generated/java/org/apache/camel/component/qute/QuteComponentConfigurer.java

```diff
@@ -0,0 +1,29 @@
+/* Generated by camel build tools - do NOT edit this file! */
```

Comment:
Look like auto-generated files should include ASF header.

Reply:
We excluded them from the checkstyle.